front/src/views/QrReader/QrReaderMultiBoxAssignToShipment.test.tsx

```diff
@@ -378,7 +371,9 @@ it("3.4.5.11 - One Box of two or more Boxes fail for the Assign boxes to shipmen
 
   // click link to remove all not failed boxes
   await user.click(screen.getByText(/Click here to remove all failed boxes from the list/i));
-  expect(await screen.findByText(/boxes selected: 0/i)).toBeInTheDocument();
-  expect(screen.queryByRole("alert")).not.toBeInTheDocument();
-  expect(screen.queryByRole("button", { name: /assign all/i })).not.toBeInTheDocument();
+  expect(await screen.findByText(/boxes selected: 1/i)).toBeInTheDocument();
+  expect(screen.queryByRole("button", { name: /assign all/i })).toBeInTheDocument();
+  // Alert appears because box was assigned to shipment and thus not inStock.
+  expect(screen.queryByRole("alert")).toBeInTheDocument();
+  expect(screen.queryByRole("button", { name: /assign all/i })).toBeDisabled();
```

Comment:
@HaGuesto @aerinsol Please check if these expectations make sense.

Reply:
This is just an alert and nothing to do with caching for shipments right?

Reply:
Exactly @aerinsol , this will only display an alert if the user tries to assign an already assigned box to a shipment.